test/testrunner.py

```diff
@@ -163,17 +291,21 @@ def main():
             setup_sm_benchmark_env(dlc_images, test_path)
             pytest_cmd = ["-s", "-rA", test_path, f"--junitxml={report}", "-n=auto", "-o", "norecursedirs=resources"]
             sys.exit(pytest.main(pytest_cmd))
+
         else:
             run_sagemaker_remote_tests(
                 [image for image in standard_images_list if not ("tensorflow-inference" in image and "py2" in image)]
             )
+        metrics_utils.send_test_duration_metrics(start_time)
+
     elif specific_test_type == "sagemaker-local":
         run_sagemaker_local_tests(
             [image for image in standard_images_list if not ("tensorflow-inference" in image and "py2" in image)]
         )
     else:
-        raise NotImplementedError(f"{test_type} test is not supported. "
-                                  f"Only support ec2, ecs, eks, sagemaker and sanity currently")
+        raise NotImplementedError(
+            f"{test_type} test is not supported. " f"Only support ec2, ecs, eks, sagemaker and sanity currently"
+        )
```

Comment:
I'm guessing this was done by `black`. In that case, you can actually join the two strings here, and replace the `<end_of_str1> " f"<start_of_str2>` with `<end_of_str1> <start_of_str2>`

Reply:
fixed